libs/fgdbg/src/DebugServer.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fgdbg/DebugServer.h"
+#include "iostream"
+
+namespace filament::fgdbg {
+
+DebugServer::DebugServer() {
+  std::cout << "FrameGraph debug server created" << std::endl;
```

Comment:
Use slog instead of std::cout inside libfilament. std::cout is only used in host tools.

Reply:
include `<utils/Log.h>` and use `utils::slog.d` instead of `std::cout`.